tests/nnapi/nnapi_gtest.skip.aarch64-linux.acl_cl

```diff
@@ -74,6 +74,8 @@ GeneratedTests.logical_not_dynamic_nnfw
 GeneratedTests.logical_or_broadcast
 GeneratedTests.logical_or_dynamic_nnfw
 GeneratedTests.logistic_dynamic_nnfw
+GeneratedTests.logistic_quant8_1
+GeneratedTests.logistic_quant8_2
```

Comment:
If you enable `GeneratedTests.logistic_quant8` tests on cpu backend, you should remove them from `tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu`, `tests/nnapi/nnapi_gtest.skip.armv7l-linux.acl_cl` and `tests/nnapi/nnapi_gtest.skip.x86_64-linux.cpu`

Reply:
@ragmani  `tests/nnapi/nnapi_gtest.skip.armv7l-linux.acl_cl` -> `tests/nnapi/nnapi_gtest.skip.armv7l-linux.cpu`?

This PR seems no problem.

Reply:
I mean there is no reason to add the tests in file. And the tests should be removed from `tests/nnapi/nnapi_gtest.skip.armv7l-linux.cpu` to verify implemented code.

Reply:
I am going to remove the test to skip in that files

Reply:
Please do not add these tests to `nnapi_gtest.skip.*.acl_cl` and `nnapi_gtest.skip.*.acl_neon`